packages/kbn-ftr-common-functional-services/services/retry/retry_for_success.ts

```diff
@@ -92,7 +92,7 @@ export async function retryForSuccess<T>(log: ToolingLog, options: Options<T>) {
     if (lastError && onFailureBlock) {
       const before = await runAttempt(onFailureBlock);
       if ('error' in before) {
-        log.debug(`--- onRetryBlock error: ${before.error.message}`);
+        log.debug(`--- onRetryBlock error: ${before.error.message} - Attempt #: ${attemptCounter}`);
```

Comment:
Could you share a `before vs. after` snapshot of logs to better understand how the failure will look like?

Reply:
I'll add it to the description!